packs/iconics/ezren-level-1.json

```diff
@@ -47,11 +188,90 @@
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "choices": [
+                            {
+                                "label": "PF2E.SkillAcr",
+                                "value": "acr"
+                            },
+                            {
+                                "label": "PF2E.SkillArc",
+                                "value": "arc"
+                            },
+                            {
+                                "label": "PF2E.SkillAth",
+                                "value": "ath"
+                            },
+                            {
+                                "label": "PF2E.SkillCra",
+                                "value": "cra"
+                            },
+                            {
+                                "label": "PF2E.SkillDec",
+                                "value": "dec"
+                            },
+                            {
+                                "label": "PF2E.SkillDip",
+                                "value": "dip"
+                            },
+                            {
+                                "label": "PF2E.SkillItm",
+                                "value": "itm"
+                            },
+                            {
+                                "label": "PF2E.SkillMed",
+                                "value": "med"
+                            },
+                            {
+                                "label": "PF2E.SkillNat",
+                                "value": "nat"
+                            },
+                            {
+                                "label": "PF2E.SkillOcc",
+                                "value": "occ"
+                            },
+                            {
+                                "label": "PF2E.SkillPrf",
+                                "value": "prf"
+                            },
+                            {
+                                "label": "PF2E.SkillRel",
+                                "value": "rel"
+                            },
+                            {
+                                "label": "PF2E.SkillSoc",
+                                "value": "soc"
+                            },
+                            {
+                                "label": "PF2E.SkillSte",
+                                "value": "ste"
+                            },
+                            {
+                                "label": "PF2E.SkillSur",
+                                "value": "sur"
+                            },
+                            {
+                                "label": "PF2E.SkillThi",
+                                "value": "thi"
+                            }
+                        ],
+                        "flag": "skill",
+                        "ignored": true,
```

Comment:
Why did this one get ignored? Just set the proficiency manually?

Reply:
No clue; I didn't set that on purpose. I'll reload the branch and review this.

Reply:
I didn't mess with any of the REs that I'm aware of.

Reply:
Ah; yes, I set the proficiency manually since the PDF does not indicate which skill was selected for the Skilled Heritage. I did the same for Valeros.